docs/writing-stories/index.mdx

```diff
@@ -100,6 +100,103 @@ components/
 
 {/* prettier-ignore-end */}
 
+<If renderer="svelte">
+
+<details>
+<summary>Alternative story definitions</summary>
+ 
+For most cases, we recommend Svelte CSF users define their stories as above, which uses the default rendering of the component. However, there are alternatives to this approach.
+
+Any `children` elements passed to the Story component will be passed along as `args.children` to the component:
+
+```svelte title="Alert.stories.svelte"
+{/* Renders <Alert>Alert text</Alert> */}
+<Story name="Alert with children">
+  Alert text
+</Story>
+```
+
+If you need to customize the rendering of your story, you can provide a snippet that accepts `args` and renders whatever you need.
```

Comment:
1. it also accepts `storyContext` as the second argument.
2. should you maybe write out explicitly that the snippet is called `template`?